cmd/jaeger/internal/integration/query_test.go

```diff
@@ -26,10 +26,6 @@ func TestJaegerQueryService(t *testing.T) {
 		ConfigFile:      "../../config-remote-storage-backend.yaml",
 		HealthCheckPort: 12133,
 		MetricsPort:     8887,
-		EnvVarOverrides: map[string]string{
-			"JAEGER_QUERY_GRPC_ENDPOINT": "localhost:0",
-			"JAEGER_QUERY_HTTP_ENDPOINT": "localhost:0",
-		},
```

Comment:
Why isn't this needed? The main collector config has query in it, and query config also has it, they would cause conflicts.

Reply:
@yurishkuro This is using the remote storage backend which removes the query service

Reply:
ah... we definitely need the diagrams, you changed how query test is architected - the "backend" acts as both collector and the remote storage (clever!)

Reply:
Hahah yeah I had to move those around in the last PR. And yes absolutely - will address in  https://github.com/jaegertracing/jaeger/issues/7055